src/components/CopyPaste/Strings.ts

```diff
@@ -0,0 +1,6 @@
+import { Translation, Language } from '../../utils/translation';
+
+Translation.register(Language.English, {
+    CopyPaste_copy: 'Copy',
+    CopyPaste_copied: 'Copied !'
```

Comment:
```suggestion
    CopyPaste_copied: 'Copied!'
```

https://www.btb.termiumplus.gc.ca/tcdnstyl-chap?lang=eng&lettr=chapsect7&info0=7